spec.html

```diff
@@ -23295,6 +23394,10 @@ <h1>Forbidden Extensions</h1>
       <li>
         When parsing for the |Module| goal symbol, the lexical grammar extensions defined in <emu-xref href="#sec-html-like-comments"></emu-xref> must not be supported.
       </li>
+      <li>
+        <ins>|ImportCall| must not be extended.</ins>
+        <!-- This is so that in the future we can potentially add new arguments or support ArgumentList. -->
```

Comment:
Given this intended support, should a trailing comma be allowed in the grammar? (not an editor question, just a curious delegate question)

Reply:
Hmm, I guess I have to say that, personally, I would expect a trailing comma to be allowed, not because of potential future extensions, but just because we allow trailing commas in ordinary calls. However, I think we should pursue this possible change in a separate needs-consensus PR, as the grammar in this proposal is already widely implemented and shipped, and it was promoted to Stage 3 with the interaction visible to the committee.